src/control/lib/control/pool.go

```diff
@@ -902,6 +904,11 @@ type (
 		// TargetsDisabled is the number of inactive targets in pool.
 		TargetsDisabled uint32 `json:"targets_disabled"`
 
+		// Latest pool global version
+		UpgradeLayoutVer uint32 `json:"upgrade_layout_ver"`
```

Comment:
On a Go-specific note, the comments on these member variables are used to create documentation. But to do that, they need to start with the variable name. Please see the examples elsewhere in this file.

As an aside, I find "UpgradeLayoutVersion" a bit confusing personally especially next to a comment calling it something totally different, not mentioning upgrades at all.